projects/plugins/jetpack/_inc/client/state/site-products/reducer.js

```diff
@@ -34,18 +34,46 @@ export const reducer = combineReducers( {
 /**
  * Returns true if currently requesting site products. Otherwise false.
  *
- * @param  {Object}  state Global state tree
- * @return {Boolean}       Whether site products are being requested
+ * @param   {object} state - Global state tree
+ * @returns {boolean}        Whether site products are being requested
  */
 export function isFetchingSiteProducts( state ) {
 	return !! state.jetpack.siteProducts.requests.isFetching;
 }
 
 /**
  * Returns WP.com site products that are relevant to Jetpack.
- * @param  {Object}  state Global state tree
- * @return {Object}  Site products
+ *
+ * @param   {object} state -  Global state tree
+ * @returns {object} Site products
  */
 export function getSiteProducts( state ) {
 	return state.jetpack.siteProducts.items;
 }
+
+/**
+ * Returns a Jetpack product if it exists in the state.
+ *
+ * @param   {object} state - Global state tree
+ * @param   {string} slug  - Product slug
+ * @returns {object} Product
+ */
+export function getSiteProduct( state, slug ) {
+	return state.jetpack.siteProducts.items?.[ slug ];
+}
+
+/**
+ * Returns the monthly cost of a product. This also takes into account intro offers.
+ *
+ * @see     pbNhbs-53E-p2
+ * @todo    Fix how we calculate the price in the future.
+ * @param   {object} state - Global state tree
+ * @param   {string} slug  - Product slug
+ * @returns {number}  Monthly cost of a product
+ */
+export function getSiteProductMonthlyCost( state, slug ) {
```

Comment:
I can't remember exactly how they work in the pricing object, but do we need to account for potential sale coupons in this function too? I know this is a temporary solution until we are able to get the pricing where we want it, just a thought I had

Reply:
Good thought. This follows the same formula as the other places where it calculates the price, and IIRC it takes the sale coupons into account.